Source/DataSources/BillboardVisualizer.js

```diff
@@ -123,7 +128,9 @@ define([
             if (!defined(billboard)) {
                 var billboardCollection = this._billboardCollection;
                 if (!defined(billboardCollection)) {
-                    billboardCollection = new BillboardCollection();
+                    billboardCollection = new BillboardCollection({
```

Comment:
This can all be written as:
```javascript
this._billboardCollection = this._scene.primitives.add(new BillboardCollection({
    scene : this._scene
}));
```